aiofcm/connection.py

```diff
@@ -161,21 +196,27 @@ def is_busy(self):
 class FCMConnectionPool:
     MAX_ATTEMPTS = 10
 
-    def __init__(self, sender_id, api_key, max_connections=10, loop=None):
+    def __init__(self, sender_id, api_key, callback=None, min_connections=0, max_connections=10, loop=None):
         # type: (int, str, int, Optional[asyncio.AbstractEventLoop]) -> NoReturn
         self.sender_id = sender_id
         self.api_key = api_key
+        self.callback = callback
+        assert min_connections <= max_connections, "min_connections is greater than max_connections"
+        self.min_connections = min_connections
         self.max_connections = max_connections
         self.loop = loop or asyncio.get_event_loop()
         self.connections = []
         self._lock = asyncio.Lock(loop=self.loop)
 
         self.loop.set_exception_handler(self.__exception_handler)
 
+        asyncio.ensure_future(self.maintain_min_connections_open())
```

Comment:
Let's save this task in the pool and cancel it in `close` method.

Reply:
Done